extensions/amp-fx-scroll-sync/0.1/scroll-sync-effect.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export class ScrollSyncEffect {
+  constructor(element) {
+    this.element = element;
+  }
+
+  isDirectional() {
+    return false;
+  }
+
+  measure() {
+
+  }
+
+  requestMeasure() {
```

Comment:
I'm not sure what this method does according to the design doc. @dvoytenko When do we want to request measure?